paper-input-behavior.html

```diff
@@ -384,7 +384,16 @@
         label.id = labelledBy;
       }
       this._ariaLabelledBy = labelledBy;
-    }
+    },
+
+    _onChange: function(event) {
+      // stop the propagation in shady DOM
+      event.stopPropagation();
```

Comment:
Is this required? We backed away a similar strategy in `IronControlState` because it causes unexpected event propagation behavior.